css/wishList.css

```diff
@@ -0,0 +1,46 @@
+@import url("/styles.css");
+
+.main-wrapper{
+    display: flex;
+    flex-direction: column;
+}
+
+.main-wrapper h1{
+    margin: 1rem;
+    align-items: center;
+    justify-content: center;
+}
+
+.wishlist-wrapper{
+    flex-grow: 1;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(260px, 1fr));
+    margin: 1rem;
+    padding: 1rem;
+}
+
+.card-comp{
+    width: auto;
+}
+
+.footer-card{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+.btn-addToCart{
+    margin: 0.625rem;
+    background-color: orange;
```

Comment:
Suggestion to consider defining this color as a custom color in a variable instead
As each browser might have their own color code definition for `orange`

Reply:
okay noted.